src/rust/graph-query-service/Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [package]
-name = "graph-query-service"
+name = "graph-query"
```

Comment:
Yeah - I don't want `-service` showing up anywhere, so I removed it in this context. You'll notice the directory name is still `-service`. I'll change that in an upcoming PR.